angularjs-portal-home/src/main/webapp/my-app/marketplace/partials/marketplace-details.html

```diff
@@ -52,7 +52,13 @@ <h3 tabindex="0" class="md-title">Description</h3>
                     </div>
                     <div class="desc-item">
                         <h3 tabindex="0" class="md-title">Ratings</h3>
-                        <span>
+                        <span ng-if="$storage.linkRatingsApi">
+                            <rating ng-model="portlet.rating" readonly="true" class="rating"></rating>
+                            
+                            ( <a ng-href="/portal/api/marketplace/{{portlet.fname}}/ratings">{{::portlet.userRated}} rating<span ng-if="portlet.userRated !== 1">s</span></a> )
```

Comment:
I don't see how to lay hands on `SERVICE_LOC` from this template? Would the deeper solution be to make `ratingApiUrl` a property of `portlet` so it'd be `{portlet.ratingApiUrl}`?